cnvlib/commands.py

```diff
@@ -86,6 +86,12 @@ def public(fn):
 AP.set_defaults(func=lambda args: AP.print_help())
 AP_subparsers = AP.add_subparsers(help="Sub-commands (use with -h for more info)")
 
+# Shared parameters
+def add_argument_treat_par_on_chrx_as_autosomal_for_genome_build(P):
+    P.add_argument('--treat-par-on-chrx-as-autosomal-for-genome-build',
```

Comment:
Should `P` be `parser`? I see an urge to refactor commands.py more substantially this way and collect other reused options into functions like this, but it can be left for the future.

I see help is "todo" here -- how about: "Treat the PAR region of human chromome X as autosomal using its known reference genome coordinates."